src/snowflake/snowpark/session.py

```diff
@@ -742,10 +744,10 @@ def add_packages(
                 is supported as a `version specifier <https://packaging.python.org/en/latest/glossary/#term-Version-Specifier>`_
                 for this argument. If a ``module`` object is provided, the package will be
                 installed with the version in the local environment.
-            force_push: Force upload unavailable Python packages with native dependencies.
-            force_install: Ignores environment present on persist_path and overwrites it with a fresh installation.
+            force_push: Force upload unavailable Python packages with native dependencies (experimental).
```

Comment:
"native" dependency might be confusing. People may think of native c code. Do you mean "local"?

Reply:
No I mean native C code, will rewrite this.